pkg/registry/core/shoot/storage/tableconvertor.go

```diff
@@ -92,6 +93,11 @@ func (c *convertor) ConvertToTable(ctx context.Context, obj runtime.Object, tabl
 		} else {
 			cells = append(cells, "<none>")
 		}
+		if shoot.Spec.Hibernation != nil && shoot.Spec.Hibernation.Enabled != nil {
+			cells = append(cells, *shoot.Spec.Hibernation.Enabled)
```

Comment:
Do you want to show the various states as well?

Hibernated -> `.spec.hibernation.enabled && .status.hibernated`
Hibernating -> `.spec.hibernation.enabled && !.status.hibernated`
Awaking -> `!.spec.hibernation.enabled && .status.hibernated`
Awake -> `!.spec.hibernation.enabled && !.status.hibernated`

Reply:
Good idea, even better. Done 👍